New Left Review.js

```diff
@@ -0,0 +1,257 @@
+{
+	"translatorID": "b97af278-52c1-4fc4-9c8b-5175c83941f2",
+	"label": "New Left Review",
+	"creator": "Bo An",
+	"target": "^https?://(www.)?newleftreview.org/issues/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-06-12 07:59:12"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	Copyright © 2021 Bo An
+	This file is part of Zotero.
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+	***** END LICENSE BLOCK *****
+*/
+
+
+function detectWeb(doc, url) {
+	const article = "articles";
+	const isArticle = url.includes(article);
+	if (isArticle) {
+		return 'journalArticle';
+	}
+	else {
+		return 'multiple';
+	}
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), (items) => {
+			if (!items) {
+				return true;
+			}
+			const articles = [];
+			for (const i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+			return true;
+		});
+	}
+	else {
+		scrape(doc, url);
+	}
+}
+
+function getSearchResults(doc) {
+	let items = {};
+	let found = false;
+
+	const articleLinks = ZU.xpath(doc, '//a[@class="article-link"]');
+	const titles = ZU.xpath(doc, '//span[@class="programme-note__title"]');
+	const authors = ZU.xpath(doc, '//span[@class="programme-note__author"]');
+
+	articleLinks.forEach((articleLink, articleIndex) => {
```

Comment:
It looks to me like the `.programme-note__title` and `.programme-note__author` elements are children of the `a.article-link` for each article. You could remove lines 68-69...